File: 6. Python/EX37.py
# Solicita a quantidade de casos a serem digitados
qtd = int(input("# Quantos casos voce vai digitar: "))

# Loop para cada caso
for i in range(qtd):

    som = 0  # Inicializa a soma
    print("")  # Linha em branco para separar casos

    # Loop para entrada de notas
    for j in range(3):

        n = float(input(f"# Nota {j+1}: "))  # Lê a nota

        # Aplica os pesos às notas
        if j == 0:
            som += n * 2  # Peso 2 para a primeira nota

        elif j == 1:
            som += n * 3  # Peso 3 para a segunda nota
            
        else:
            som += n * 5  # Peso 5 para a terceira nota

    # Calcula e exibe a média
    print(f"# Media = {(som / 10):.1f}")  # Média ponderada

File: 4. Java/EX37.java
package Apostila;

import java.util.Locale;
import java.util.Scanner;

public class EX37
{
    public static void main(String[] args)
    {
        // Variáveis
        int qtd;
        double n, som;

        // Define o locale para formato de números
        Locale.setDefault(Locale.US);
        
        // Scanner para entrada
        try (Scanner ent = new Scanner(System.in))
        {
            // Lê a quantidade de casos
            System.out.print("# Quantos casos voce vai digitar: ");
            qtd = ent.nextInt();

            // Laço para cada caso
            for (int i = 0; i < qtd; i++)
            {
                // Reinicia a soma para cada caso
                som = 0;

                // Laço para ler 3 notas
                for (int j = 0; j < 3; j++)
                {
                    System.out.printf("# Nota [%d]: ", j+1);                    
                    n = ent.nextDouble();

                    // Aplica pesos às notas
                    if (j == 0)
                    {
                        som += n * 2;
                    }

                    else if (j == 1)
                    {
                        som += n * 3;
                    }

                    else
                    {
                        som += n * 5;
                    }
                }
                
                // Calcula e exibe a média
                System.out.println("# Media = " + String.format("%.1f", (double) som/10) + "\n");
            }
        }
    }
}
